service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -97,52 +132,65 @@ public PolicyEvaluationResult evalPolicy(final PolicyEvaluationRequestV1 request
             } else {
                 supplementalSubjectAttributes = new HashSet<>(request.getSubjectAttributes());
             }
-            result = evalPolicy(request.getResourceIdentifier(), request.getSubjectIdentifier(), request.getAction(),
-                    supplementalResourceAttributes, supplementalSubjectAttributes);
+
+            for (PolicySet policySet : filteredPolicySets) {
+                result = evalPolicySet(policySet, subjectIdentifier, uri, action, supplementalResourceAttributes,
+                        supplementalSubjectAttributes);
+                if (result.getEffect() != Effect.NOT_APPLICABLE) {
+                    break;
+                }
+                // Continue to the next policy set if evaluation result is NOT_APPLICABLE
+            }
+
+            LOGGER.info(
+                    String.format(
+                            "Processed Policy Evaluation for: "
+                                    + "resourceUri=[%s], subjectIdentifier=[%s], action=[%s]," + " result=[%s]",
```

Comment:
Once again. Typically we use "'" for delimiters in log messages. If we're switching to "[", why is that better and shouldn't we change that everywhere for consistency? My key concern here is consistency.